docs/hub/model-card-annotated.md

```diff
@@ -223,7 +223,7 @@ _What do the results say? This can function as a kind of tl;dr for general audie
 
 **Section Overview:** Summarizes the information necessary to calculate environmental impacts such as electricity usage and carbon emissions.
 
-* **Hardware Type:** `hardware`
+* **Hardware Type:** `hardware_type`
```

Comment:
This will be a (small) breaking change for https://github.com/huggingface/huggingface_hub/blob/main/src/huggingface_hub/templates/modelcard_template.md cc @Wauplin . Are we fine with moving this forward?

Reply:
I think it's fine yes. I'd better have a clean template even if it means introducing a small breaking change rather than handling backward compatibility (at the cost of worsen readability). But thanks for the ping!